pay/pubspec.yaml

```diff
@@ -18,8 +18,8 @@ version: 1.1.0
 repository: https://github.com/google-pay/flutter-plugin/tree/main/pay
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
```

Comment:
Let's keep single quotes in these two lines.

Reply:
Fixed in 5bfa1a6389dd13c70f4b4982db7f9d02ffa176b3